apps/bookmark/js/bookmark_editor.js

```diff
@@ -28,6 +29,8 @@ var BookmarkEditor = {
     this.cancelButton = document.getElementById('cancel-button');
     this.saveButton = document.getElementById('done-button');
     this.appInstallationSection = document.getElementById('app-installation');
+    this.appIcon = document.getElementById('app-icon');
```

Comment:
Up to you, although I would mote these vars to https://github.com/mozilla-b2g/gaia/pull/29024/files#diff-32abdeb6061956598cbede801aa9bc1aR86 and https://github.com/mozilla-b2g/gaia/pull/29024/files#diff-32abdeb6061956598cbede801aa9bc1aR91 respectively